src/com/codersbay/Stack.java

```diff
@@ -0,0 +1,64 @@
+package com.codersbay;
+
+import java.util.ArrayList;
+
+public class Stack {
+
+    public ArrayList<Integer> myStack;
+
+    public Stack(ArrayList<Integer> myStack) {
+        this.myStack = myStack;
+    }
+
+    public void push (Integer newElement){
+        this.myStack.add(newElement);
+    }
+
+    public int size(){
+        return this.myStack.size();
+    }
+
+    public int pop(){
+        int maxIndex = this.myStack.size() - 1;
+        int maxNumb =  this.myStack.get(maxIndex);
+        this.myStack.remove(maxIndex);
```

Comment:
`remove` würde dir bereits das Element an der Stelle zurückgeben, also müsstest du gar nicht explizit get vorher aufrufen 😉 

Was hier noch fehlt ist die `StackTooSmallException` 😭 die solltest du in der pop und peek Methode werfen, wenn der Stack zu klein ist also in der pop() und peek() wenn er leer ist, in der pop(int n) methode wenn keine n Elemente da sind.